build-system/tasks/clean.js

```diff
@@ -61,6 +61,10 @@ async function clean() {
   if (argv.include_subpackages) {
     pathsToDelete.push('**/node_modules', '!node_modules');
   }
+  if (argv.exclude) {
+    const excludes = argv.exclude.split(',');
+    pathsToDelete = pathsToDelete.filter((path) => !excludes.includes(path));
```

Comment:
Instead of doing an exact match and risking a mismatch due to a stray slash, might be safer to explicitly negate these paths.
```suggestion
    for (const exclude of excludes) {
      pathsToDelete.push(`!${exclude}`);
    }
```